design/schema/bcdc.puml

```diff
@@ -0,0 +1,319 @@
+@startuml
+
+hide empty members
+'hide circle
+
+
+'Enums
+enum WebResourceType {
+    Download
+    Information
+    Explore
+    View
+}
+
+enum AccessControl {
+    Open
+    Embargoed
+    Controlled
+}
+
+enum CompletionState {
+    Complete
+    InProgress
+}
+
+enum AgentType {
+    Person
+    Organization
+}
+
+
+'Entity-like
+class Agent {
+    * id: String!
+    agent_id: String!
+    agentType: AgentType
+    organization: Organization
+    person: Person
+    type: AgentType!
+}
+
+class Modality {
+    * id: String!
+    name: String!
+    handle: String
+}
+
+class SpecimenType {
+    * id: String!
+    name: String!
+    handle: String
+}
+
+
+class WebResource {
+    * id: String!
+    handle: String!
```

Comment:
what is handle?  This doesn't seem like a typical field name. Should be webresource "name"  maybe?

Reply:
Changed handle to name